packages/aws-cdk-lib/aws-opensearchservice/test/domain.test.ts

```diff
@@ -2676,3 +2676,94 @@ function testMetric(
   });
   expect(metric.dimensions).toHaveProperty('DomainName');
 }
+
+each(testedOpenSearchVersions).test('can configure coordinator nodes with nodeOptions', (engineVersion) => {
```

Comment:
can you add a test case where we have two NodeOptions items and the second one is has invalid type that doesn't end with `.search`.

Reply:
We can't have two nodeoptions of same node type